cmd/lakectl/cmd/local.go

```diff
@@ -25,6 +25,14 @@ const (
 	localDefaultMaxArgs         = 1
 )
 
+const localSummaryTemplate = `
+{{ if and (eq .Downloaded 0) (eq .Removed 0) (eq .Uploaded 0)}}No changes{{else}}{{if 
+gt .Downloaded 0 }}{{"Downloaded:" | printf|green}} {{.Downloaded|green}}{{end}}{{if 
+gt .Uploaded 0 }}{{"\nUploaded:" | printf|yellow}} {{.Uploaded|yellow}}{{end}}{{if 
+gt .Removed 0 }}{{"\nRemoved:" | printf|red}} {{.Removed|red}}{{end}}
+{{end}}
```

Comment:
1. Consider adding `Summary` or `{{ .Operation }} Summary }}` as first line. Each operations just print it out before rendering the template. Ex: for checkout we can call `Write(localSummaryTemplate, "Checkout", syncMgr.Summary())` and it will render `\nCheckout Summary\n...` (\n here is just to render it in one line.
2. No need to check each one for `gt` - we like "No changes." or render all numbers, right?

Reply:
I'll adopt your approach